test/dummy.test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+/* global suite: false, setup: false, test: false,
+    teardown: false, suiteSetup: false, suiteTeardown: false */
+const assert = require('assert');
+const common = require('./common');
+
+
+suite('dummy', () => {
```

Comment:
I think this might be a problem. Mocha does some very fancy things with `this`, hence using fat arrow functions will remove the ability to set things like `this.timeout(10000);`, etc.

Therefore only for `suite` --->
```
suite('dummy', function() {
...
});
```